File: fizzbuzz.f90
! Language:       Fortran 90
! Web site:       https://gcc.gnu.org/wiki/GFortran (for the GNU implementation)
! Last tested on: Ubuntu 22.04.5
! Requires:       apt-get install gfortran

       program FizzBuzz
       implicit none
       integer i

       do i = 1, 100

           if (mod(i, 15) .eq. 0) then
               print '(a)', 'FizzBuzz'
           else if (mod(i, 3) .eq. 0) then
               print '(a)', 'Fizz'
           else if (mod(i, 5) .eq. 0) then
               print '(a)', 'Buzz'
           else if (i .lt. 10) then
               print '(i1)', i
           else
               print '(i2)', i
           end if

       end do

       end

File: fizzbuzz.pas
(*
 * Language:       Pascal
 * Web site:       http://www.freepascal.org/
 * Last tested on: Ubuntu 18.04
 * Requires:       apt-get install fp-compiler
 *)
program FizzBuzz;
var
    I: Integer;
begin
    for I := 1 to 100 do
    begin
        if I mod 15 = 0 then
            writeln('FizzBuzz')
        else if I mod 3 = 0 then
            writeln('Fizz')
        else if I mod 5 = 0 then
            writeln('Buzz')
        else
            writeln(I);
    end;
end.
